imports/collections/schemas/cart.js

```diff
@@ -245,55 +241,14 @@ export const Cart = new SimpleSchema({
   "shipping.$": {
     type: Shipment
   },
+  /* Working to get rid of cart.billing, but currently still where discounts are applied to carts */
   "billing": {
     type: Array,
     optional: true
   },
   "billing.$": {
     type: Payment
   },
-  "tax": {
```

Comment:
Yes, it is only stored in the Cart collection as `tax` and `taxRate` on each item, and these are reduced to get total tax in GraphQL resolvers and Meteor UI. I did not want to risk a mismatch by storing the totals in the db since they can be easily calculated. It's not an issue in Order because the items and quantities don't change once it's an order.